test/ad/utils.jl

```diff
@@ -47,38 +51,43 @@ function test_ad(f, x, broken=(); rtol=1e-6, atol=1e-6)
         end
     end
 
-    # TODO(mhauru) The version bound should be relaxed once some Enzyme issues get
-    # sorted out. I think forward mode will remain broken for versions <= 1.6 due to
-    # some Julia bug. See https://github.com/EnzymeAD/Enzyme.jl/issues/1629 and
-    # discussion in https://github.com/TuringLang/Bijectors.jl/pull/318.
-    if (AD == "All" || AD == "Enzyme") && VERSION >= v"1.10"
+    if AD == "All" || AD == "Enzyme"
         forward_broken = :EnzymeForward in broken || :Enzyme in broken
         reverse_broken = :EnzymeReverse in broken || :Enzyme in broken
-        if forward_broken
-            @test_broken(
-                collect(et, Enzyme.gradient(Enzyme.Forward, f, x)) ≈ finitediff,
-                rtol = rtol,
-                atol = atol
-            )
-        else
-            @test(
-                collect(et, Enzyme.gradient(Enzyme.Forward, f, x)) ≈ finitediff,
-                rtol = rtol,
-                atol = atol
-            )
+        if !(:EnzymeForwardCrash in broken)
+            if forward_broken
+                @test_broken(
+                    collect(et, Enzyme.gradient(Enzyme.Forward, f, x)[1]) ≈ finitediff,
```

Comment:
this collect shouldn't be required anymore

Reply:
Great, I removed it 🙂